runtime/vm/JFRChunkWriter.cpp

```diff
@@ -863,4 +863,14 @@ VM_JFRChunkWriter::writeInitialEnvironmentVariableEvents()
 		}
 	}
 }
+
+void
+VM_JFRChunkWriter::writeNativeLibraryEvents()
+{
```

Comment:
we should add a OMR call to do this since we will need it for all platforms

Reply:
What would the OMR call look like?